<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.189">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Analysis showcase</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="analysis showcase_files/libs/clipboard/clipboard.min.js"></script>
<script src="analysis showcase_files/libs/quarto-html/quarto.js"></script>
<script src="analysis showcase_files/libs/quarto-html/popper.min.js"></script>
<script src="analysis showcase_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="analysis showcase_files/libs/quarto-html/anchor.min.js"></script>
<link href="analysis showcase_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="analysis showcase_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="analysis showcase_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="analysis showcase_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="analysis showcase_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

<script src="analysis showcase_files/libs/kePrint-0.0.1/kePrint.js"></script>
<link href="analysis showcase_files/libs/lightable-0.0.1/lightable.css" rel="stylesheet">

  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul class="collapse">
  <li><a href="#overview." id="toc-overview." class="nav-link active" data-scroll-target="#overview.">Overview.</a></li>
  <li><a href="#background" id="toc-background" class="nav-link" data-scroll-target="#background">Background</a>
  <ul class="collapse">
  <li><a href="#within-species-only" id="toc-within-species-only" class="nav-link" data-scroll-target="#within-species-only">Within species only</a></li>
  <li><a href="#within-between-species" id="toc-within-between-species" class="nav-link" data-scroll-target="#within-between-species">Within + between species</a></li>
  <li><a href="#between-species-only" id="toc-between-species-only" class="nav-link" data-scroll-target="#between-species-only">Between species only</a></li>
  </ul></li>
  <li><a href="#phylogenies-work-in-progress" id="toc-phylogenies-work-in-progress" class="nav-link" data-scroll-target="#phylogenies-work-in-progress">Phylogenies (work in progress)</a></li>
  <li><a href="#predicting-coexistence-trajectories-across-species" id="toc-predicting-coexistence-trajectories-across-species" class="nav-link" data-scroll-target="#predicting-coexistence-trajectories-across-species">Predicting coexistence trajectories across species</a></li>
  <li><a href="#climatic-temperature" id="toc-climatic-temperature" class="nav-link" data-scroll-target="#climatic-temperature">Climatic Temperature</a></li>
  <li><a href="#absolute-temperature-vs-asymmetry-to-greenhouse-conditions." id="toc-absolute-temperature-vs-asymmetry-to-greenhouse-conditions." class="nav-link" data-scroll-target="#absolute-temperature-vs-asymmetry-to-greenhouse-conditions.">Absolute temperature vs asymmetry to greenhouse conditions.</a></li>
  <li><a href="#temperature-x-genetic" id="toc-temperature-x-genetic" class="nav-link" data-scroll-target="#temperature-x-genetic">Temperature x Genetic</a>
  <ul class="collapse">
  <li><a href="#selected-sites-and-neutral-sites." id="toc-selected-sites-and-neutral-sites." class="nav-link" data-scroll-target="#selected-sites-and-neutral-sites.">Selected sites and “neutral” sites.</a></li>
  <li><a href="#wgs-within-s.-polyrhiza" id="toc-wgs-within-s.-polyrhiza" class="nav-link" data-scroll-target="#wgs-within-s.-polyrhiza">WGS (within <em>S. polyrhiza</em>)</a></li>
  <li><a href="#wgs-among-s.-polyrhiza-s.-intermedia" id="toc-wgs-among-s.-polyrhiza-s.-intermedia" class="nav-link" data-scroll-target="#wgs-among-s.-polyrhiza-s.-intermedia">WGS (among <em>S. polyrhiza</em> &amp; <em>S. intermedia</em>)</a></li>
  </ul></li>
  <li><a href="#temperature-x-spatial-distance" id="toc-temperature-x-spatial-distance" class="nav-link" data-scroll-target="#temperature-x-spatial-distance">Temperature x Spatial distance</a>
  <ul class="collapse">
  <li><a href="#spatial-within-s.-polyrhiza" id="toc-spatial-within-s.-polyrhiza" class="nav-link" data-scroll-target="#spatial-within-s.-polyrhiza">Spatial within <em>S. polyrhiza</em></a></li>
  </ul></li>
  <li><a href="#explaining-fitness-difference" id="toc-explaining-fitness-difference" class="nav-link" data-scroll-target="#explaining-fitness-difference">Explaining fitness difference</a>
  <ul class="collapse">
  <li><a href="#reason-1-extreme-cases" id="toc-reason-1-extreme-cases" class="nav-link" data-scroll-target="#reason-1-extreme-cases">Reason 1: Extreme cases</a></li>
  <li><a href="#reason-2-not-enough-time-for-competition." id="toc-reason-2-not-enough-time-for-competition." class="nav-link" data-scroll-target="#reason-2-not-enough-time-for-competition.">Reason 2: Not enough time for competition.</a></li>
  <li><a href="#reason-3-selection-at-different-stages" id="toc-reason-3-selection-at-different-stages" class="nav-link" data-scroll-target="#reason-3-selection-at-different-stages">Reason 3: Selection at different stages</a></li>
  <li><a href="#reason-4-not-capturing-moment-of-divergence" id="toc-reason-4-not-capturing-moment-of-divergence" class="nav-link" data-scroll-target="#reason-4-not-capturing-moment-of-divergence">Reason 4: Not capturing moment of divergence</a></li>
  </ul></li>
  </ul>
</nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Analysis showcase</h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<div class="cell">

</div>
<div class="cell">

</div>
<div class="cell">

</div>
<div class="cell">

</div>
<div class="cell">

</div>
<div class="cell">

</div>
<section id="overview." class="level1">
<h1>Overview.</h1>
<p>In this document I go over a few analyses and figures with some opinions along the way.</p>
<ol type="1">
<li>With WGS pairwise distances provided by Duchen, we can now analyse the differences between and within coexistence and create predictions based off of each.</li>
<li>As well this data allows us to construct phylogenies. I have constructed a few here but details needed at nodes would be useful (and of course to make them look more aesthetically pleasing).</li>
<li>WGS to <em>S. intermedia</em> also allows us to incorporate pairwise temperature differences with more power, given the residuals to WGS distances. The question <em>if</em> we should do this is more nuanced.</li>
<li>I also provide a few Anovas to contrast the effects of temperature, selected sites, neutral sites, WGS (among and within) and spatial to ND, FD, Coex. Results for selected and “neutral” sites are qualitatively similar to WGS so I do not provide figures though see comment re: “neutral”. Temperature appears to play a role in ND and not FD which is very interesting.</li>
<li>Finally I argue reasons for our negative slope and &gt;0 intercept of FD. If any of these sound convincing they may warrant deeper analysis.</li>
</ol>
</section>
<section id="background" class="level1">
<h1>Background</h1>
<section id="within-species-only" class="level2">
<h2 class="anchored" data-anchor-id="within-species-only">Within species only</h2>
<p>As a reminder this was about where we left off:</p>
<ul>
<li><p>ND increases over WGS intraspecific pairwise distance</p></li>
<li><p>FD decreases over WGS intraspecific pairwise distance (though with the correction I mention below, p value goes to 0.08 if that matters, I wouldn’t hold too much weight over it until bayesian).</p></li>
<li><p>Coex increases over WGS intraspecific pairwise distance</p></li>
</ul>
<div class="cell">

</div>
<div class="cell">
<div class="cell-output-display">
<p><img src="analysis-showcase_files/figure-html/unnamed-chunk-8-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="within-between-species" class="level2">
<h2 class="anchored" data-anchor-id="within-between-species">Within + between species</h2>
<p>Trends that change</p>
<ul>
<li><p>[same] ND increases over WGS pairwise distance &lt;- expected by theory</p></li>
<li><p>[different] FD <u><strong>increases</strong></u> over WGS pairwise distance &lt;- expected by theory</p></li>
<li><p>[same] Coex increases over WGS intraspecific pairwise distance &lt;- still unexpected(?) by theory under brownian motion.</p></li>
</ul>
<p>Fit under whole model, each significant. (note that this result is highly robust, even when data is ranked/compressed/logged/you name it).</p>
<div class="cell">
<div class="cell-output-display">
<p><img src="analysis-showcase_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>What if we use the within species model to predict the ND, FD, and Coex of among species?</p>
</section>
<section id="between-species-only" class="level2">
<h2 class="anchored" data-anchor-id="between-species-only">Between species only</h2>
<p>Trends that change</p>
<ul>
<li><p>[different] ND <u><strong>constant</strong></u> over WGS <strong>interspecific</strong> pairwise distance</p></li>
<li><p>[different] FD <u><strong>constant</strong></u> over WGS <strong>interspecific</strong> pairwise distance</p></li>
<li><p>[different] Coex <u><strong>constant</strong></u> over WGS <strong>interspecific</strong> pairwise distance</p></li>
</ul>
<p>For between species only, we see no significant change across genetic distances which is very interesting. Could this mean that the mechanisms driving ND and FD intraspecifically are different to those interspecifically?</p>
<div class="cell">
<div class="cell-output-display">
<p><img src="analysis-showcase_files/figure-html/unnamed-chunk-10-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
</section>
<section id="phylogenies-work-in-progress" class="level1">
<h1>Phylogenies (work in progress)</h1>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/Screenshot%202023-11-03%20at%205.13.10%20PM.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Full phylogeny based off WGS pairwise distance. Colours represent 21 groups. Will be cleaned up and made nicer and more clear which are the contrast groups.</figcaption><p></p>
</figure>
</div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/Screenshot%202023-11-03%20at%205.17.36%20PM.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Very early stages but these are our 21 phylogenies. They will of course be more legible and colour matching. I envision putting on the nodes coexistence likelihood or difference to the mean as either colouration gradient or pie charts. Would do this now but its past 5pm on friday haha</figcaption><p></p>
</figure>
</div>
</section>
<section id="predicting-coexistence-trajectories-across-species" class="level1">
<h1>Predicting coexistence trajectories across species</h1>
<p>Here we can observe how an estimation of a linear model with just within species data predicts between species GAM (with random effects). To me this is a fairly interesting result, the plateauing of niche differences with genetic distance.</p>
<ul>
<li><p>Green = linear model estimated from <em>S. polyrhiza</em> alone</p></li>
<li><p>Orange = GAM model estimated from both <em>S. polyrhiza</em> and <em>S. intermedia</em></p></li>
</ul>
<section id="niche-difference-vs-genetic-distance-wgs" class="level3">
<h3 class="anchored" data-anchor-id="niche-difference-vs-genetic-distance-wgs">Niche difference Vs Genetic distance (WGS)</h3>
<div class="cell">
<div class="cell-output-display">
<p><img src="analysis-showcase_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="fitness-difference-vs-genetic-distance-wgs" class="level3">
<h3 class="anchored" data-anchor-id="fitness-difference-vs-genetic-distance-wgs">Fitness difference Vs Genetic distance (WGS)</h3>
<p>First to do this I needed to transform the data to work.</p>
<p>I transformed the data from relative FD to log(RFD -1) specifically to prevent the model from projecting RFD &lt; 0 from the linear model (which if you recall our FD slope approaches 0 intraspecifically, suggesting negative FD inter specifically which does not make sense.)</p>
<ul>
<li>Note: Takuji had originally log transformed FD which helped but still did not quite fix normality and inadvertently allows the model to project into a domain not physically possible when projecting to <em>S. intermedia</em>.</li>
</ul>
<p>While not perfect as you can see it does help!</p>
<p>Hopefully this makes sense to everyone. (I tried other glms but they didn’t seem to play nice for this data distribution).</p>
<section id="qq-plots-for-normality-of-fd" class="level4">
<h4 class="anchored" data-anchor-id="qq-plots-for-normality-of-fd">QQ plots for normality of FD</h4>
<div class="cell">
<div class="cell-output-display">
<p><img src="analysis-showcase_files/figure-html/unnamed-chunk-12-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Now that the data is transformed, its very interesting how RFD is projected to approach 1 based off of intraspecific data. (i.e.&nbsp;our negative FD slope). I</p>
<p>Hot take perspective:</p>
<ul>
<li>I wonder if there could be an argument towards evolutionary fitness given the supposed lack of reproductive isolation represented by green.</li>
</ul>
<div class="cell">

</div>
<div class="cell">
<div class="cell-output-display">
<p><img src="analysis-showcase_files/figure-html/unnamed-chunk-14-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
</section>
<section id="coexistence-vs-genetic-distance-wgs" class="level3">
<h3 class="anchored" data-anchor-id="coexistence-vs-genetic-distance-wgs">Coexistence Vs Genetic distance (WGS)</h3>
<p>And coexistence likelihood which is perhaps the best figure!</p>
<div class="cell">

</div>
<div class="cell">
<div class="cell-output-display">
<p><img src="analysis-showcase_files/figure-html/unnamed-chunk-16-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
</section>
<section id="climatic-temperature" class="level1">
<h1>Climatic Temperature</h1>
<section id="summary-of-data-below" class="level3">
<h3 class="anchored" data-anchor-id="summary-of-data-below">Summary of data below:</h3>
<ul>
<li><p>Signal for coexistence is broadly stronger than ND or FD alone.</p></li>
<li><p>Temperature is marginally significant across selected, neutral, and WGS.</p></li>
<li><p>Neutral is marginally significant for coexistence. This may change when we get true neutral data.</p></li>
<li><p>WGS as a predictor for coex is very robust.</p></li>
</ul>
<div class="cell">

</div>
<div class="cell">
<div class="cell-output-display">
<p><img src="analysis-showcase_files/figure-html/unnamed-chunk-18-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<ul>
<li>Effect of temperature (alone) is conserved between among and within analysis.</li>
</ul>
</section>
</section>
<section id="absolute-temperature-vs-asymmetry-to-greenhouse-conditions." class="level1">
<h1>Absolute temperature vs asymmetry to greenhouse conditions.</h1>
<p>Something to consider is that performance may be dictated by distance to home environment. To quantify this we can calculate average pairwise asymmetry/difference to greenhouse conditions by:</p>
<p><span class="math inline">\(\overline T_{\:GH\:asymmetry}= abs \bigg (\dfrac{|\overline T_{i} - 21.48| - |\overline T_{j} - 21.48|}{2} \bigg )\)</span></p>
<p>Because this calculates asymmetry, if for example Ti = 31.48 and Tj = 11.48 would be just as asymmetric as both Ti and Tj = 21.48.</p>
<p>As you see below, all fairly insignificant (p&gt;0.4)</p>
<p><strong>Note</strong>: Whether we should use this or absolute temperature is debatable but I would argue absolute temperature makes more sense if we are not arguing temperature itself selects, rather everything associated with temperature may produce different niche conditions. (eg. community composition, nutrient availability, etc.)</p>
<div class="cell">

</div>
<div class="cell">
<div class="cell-output-display">
<p><img src="analysis-showcase_files/figure-html/unnamed-chunk-20-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="temperature-x-genetic" class="level1">
<h1>Temperature x Genetic</h1>
<section id="selected-sites-and-neutral-sites." class="level2">
<h2 class="anchored" data-anchor-id="selected-sites-and-neutral-sites.">Selected sites and “neutral” sites.</h2>
<section id="background-1" class="level3">
<h3 class="anchored" data-anchor-id="background-1">Background</h3>
<p>Selected sites were done in two approaches:</p>
<ul>
<li><p>Matrix selDiv has the genetic distances from all genes corresponding to the top 1% of 3P-CLR.</p></li>
<li><p>Matrix selSpWide has the distances from all genes corresponding to the top 1% of RAiSD.</p></li>
</ul>
<p>For brevity here I will show the analysis of selDiv, though results are similar.</p>
<p><strong>Neutral sites are not explicitly neutral.</strong></p>
<ul>
<li>Shuqing: <em>“The matrix”neutral” has the distances from all remaining genes. Of course this isn’t necessary “neutral”, but I could filter out more genes if you deem necessary.”</em></li>
</ul>
<p>For this analysis it does not make sense to examine between species as the scales are quite different?</p>
<p>Below is what we can see for interactive effects.</p>
<p>Note: All below analysis is done as a Type II anova</p>
<div class="cell">

</div>
<section id="selected-sites-within-s.-polyrhiza" class="level4">
<h4 class="anchored" data-anchor-id="selected-sites-within-s.-polyrhiza">Selected sites within <em>S. polyrhiza</em></h4>
<div class="cell">
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>ANOVA Selected Sites ND Results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> selDiv_value </td>
   <td style="text-align:right;"> 0.8632943 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.3528184 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 3.6667521 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0555083 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> selDiv_value:Temp_mean_diff </td>
   <td style="text-align:right;"> 0.2497082 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.6172806 </td>
  </tr>
</tbody>
</table>

</div>
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>ANOVA Selected Sites log(FD-1) results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> selDiv_value </td>
   <td style="text-align:right;"> 4.0161640 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0450661 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 0.0796628 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.7777549 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> selDiv_value:Temp_mean_diff </td>
   <td style="text-align:right;"> 0.8243062 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.3639241 </td>
  </tr>
</tbody>
</table>

</div>
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>ANOVA Selected Sites Coexistence Results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> selDiv_value </td>
   <td style="text-align:right;"> 4.8945442 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0269417 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 1.3849033 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.2392677 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> selDiv_value:Temp_mean_diff </td>
   <td style="text-align:right;"> 0.8565437 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.3547077 </td>
  </tr>
</tbody>
</table>

</div>
</div>
</section>
<section id="neutral-ish-sites-within-s.-polyrhiza" class="level4">
<h4 class="anchored" data-anchor-id="neutral-ish-sites-within-s.-polyrhiza">Neutral-ish sites within <em>S. polyrhiza</em></h4>
<div class="cell">

</div>
<div class="cell">
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>ANOVA neutral Sites ND Results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> neutral_value </td>
   <td style="text-align:right;"> 0.6109541 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.4344288 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 3.7812621 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0518296 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> neutral_value:Temp_mean_diff </td>
   <td style="text-align:right;"> 0.2092111 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.6473864 </td>
  </tr>
</tbody>
</table>

</div>
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>ANOVA neutral Sites log(FD-1) results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> neutral_value </td>
   <td style="text-align:right;"> 3.5775984 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0585638 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 0.0609151 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.8050553 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> neutral_value:Temp_mean_diff </td>
   <td style="text-align:right;"> 0.6722868 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.4122555 </td>
  </tr>
</tbody>
</table>

</div>
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>ANOVA neutral Sites Coexistence Results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> neutral_value </td>
   <td style="text-align:right;"> 3.5934809 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0580066 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 1.5251486 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.2168420 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> neutral_value:Temp_mean_diff </td>
   <td style="text-align:right;"> 0.5178036 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.4717801 </td>
  </tr>
</tbody>
</table>

</div>
</div>
</section>
</section>
</section>
<section id="wgs-within-s.-polyrhiza" class="level2">
<h2 class="anchored" data-anchor-id="wgs-within-s.-polyrhiza">WGS (within <em>S. polyrhiza</em>)</h2>
<ul>
<li>Note: with the transformation FD based of WGS becomes slightly weaker</li>
</ul>
<div class="cell">

</div>
<div class="cell">
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>ANOVA WGS ND Results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> pw_WGS </td>
   <td style="text-align:right;"> 3.9393406 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0471693 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 3.3206574 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0684145 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> pw_WGS:Temp_mean_diff </td>
   <td style="text-align:right;"> 0.4090223 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.5224662 </td>
  </tr>
</tbody>
</table>

</div>
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>ANOVA WGS log(FD-1) results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> pw_WGS </td>
   <td style="text-align:right;"> 2.9565258 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0855315 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 0.0198476 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.8879636 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> pw_WGS:Temp_mean_diff </td>
   <td style="text-align:right;"> 0.6065527 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.4360887 </td>
  </tr>
</tbody>
</table>

</div>
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>ANOVA WGS Coexistence Results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> pw_WGS </td>
   <td style="text-align:right;"> 8.926924 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0028100 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 1.623254 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.2026386 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> pw_WGS:Temp_mean_diff </td>
   <td style="text-align:right;"> 1.342556 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.2465837 </td>
  </tr>
</tbody>
</table>

</div>
</div>
</section>
<section id="wgs-among-s.-polyrhiza-s.-intermedia" class="level2">
<h2 class="anchored" data-anchor-id="wgs-among-s.-polyrhiza-s.-intermedia">WGS (among <em>S. polyrhiza</em> &amp; <em>S. intermedia</em>)</h2>
<ul>
<li>Note: with the transformation FD based of WGS becomes slightly weaker</li>
</ul>
<div class="cell">

</div>
<div class="cell">
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>Among ANOVA WGS ND Results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> pw_WGS </td>
   <td style="text-align:right;"> 167.880360 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0000000 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 8.771491 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0030598 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> pw_WGS:Temp_mean_diff </td>
   <td style="text-align:right;"> 4.365298 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0366781 </td>
  </tr>
</tbody>
</table>

</div>
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>Among ANOVA WGS log(FD-1) results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> pw_WGS </td>
   <td style="text-align:right;"> 53.3323915 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0000000 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 0.3207949 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.5711304 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> pw_WGS:Temp_mean_diff </td>
   <td style="text-align:right;"> 2.4313310 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.1189319 </td>
  </tr>
</tbody>
</table>

</div>
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>Among ANOVA WGS Coexistence Results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> pw_WGS </td>
   <td style="text-align:right;"> 37.123841 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0000000 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 7.098584 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0077145 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> pw_WGS:Temp_mean_diff </td>
   <td style="text-align:right;"> 3.995274 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0456280 </td>
  </tr>
</tbody>
</table>

</div>
</div>
<ul>
<li>because of the increased power perhaps, ND increases with temperature when including pairwise contrasts to intermedia as well. Whether we should use this or the previous data to argue for temperature depends on specifically what we are trying to say in the context of speciation. Temperature definitely appears to have an affect on niche (rather than fitness) though.</li>
</ul>
</section>
</section>
<section id="temperature-x-spatial-distance" class="level1">
<h1>Temperature x Spatial distance</h1>
<p>Similarly, for this analysis it does not make too much sense to include <em>S. intermedia</em>.</p>
<div class="cell">

</div>
<section id="spatial-within-s.-polyrhiza" class="level2">
<h2 class="anchored" data-anchor-id="spatial-within-s.-polyrhiza">Spatial within <em>S. polyrhiza</em></h2>
<div class="cell">
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>ANOVA spatial x temperature ND Results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> spatial.D </td>
   <td style="text-align:right;"> 0.0780516 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.7799553 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 4.5502165 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0329146 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> spatial.D:Temp_mean_diff </td>
   <td style="text-align:right;"> 0.9504639 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.3296013 </td>
  </tr>
</tbody>
</table>

</div>
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>ANOVA spatial x temperature log(FD-1) results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> spatial.D </td>
   <td style="text-align:right;"> 0.2576502 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.6117391 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 0.0798913 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.7774447 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> spatial.D:Temp_mean_diff </td>
   <td style="text-align:right;"> 0.2948870 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.5871057 </td>
  </tr>
</tbody>
</table>

</div>
<div class="cell-output-display">

<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>ANOVA spatial x temperature Coexistence Results</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:right;"> Chisq </th>
   <th style="text-align:right;"> Df </th>
   <th style="text-align:right;"> Pr(&gt;Chisq) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> spatial.D </td>
   <td style="text-align:right;"> 0.6262635 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.4287292 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Temp_mean_diff </td>
   <td style="text-align:right;"> 3.4386094 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.0636893 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> spatial.D:Temp_mean_diff </td>
   <td style="text-align:right;"> 0.2287548 </td>
   <td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 0.6324487 </td>
  </tr>
</tbody>
</table>

</div>
</div>
</section>
</section>
<section id="explaining-fitness-difference" class="level1">
<h1>Explaining fitness difference</h1>
<section id="reason-1-extreme-cases" class="level2">
<h2 class="anchored" data-anchor-id="reason-1-extreme-cases">Reason 1: Extreme cases</h2>
<p>I may have another hypothesis as to why we have a significant negative slope for fitness differences. I am wondering if it’s because we have very few genotypes that exist at low pair wise genetic distance. This is because most genotypes that have low genetic distance will likely have low spatial distance and are not in our design. What that means is that we may be biasing our selection of close relatives to those that are spatially very far away. Perhaps this is only achievable biologically if one of those genotypes (far from origin) is very fit. Of course theoretically there is no correlation, not negatively correlated. But if we look at the among lineages:</p>
<div class="cell">
<div class="cell-output-display">
<p><img src="analysis-showcase_files/figure-html/unnamed-chunk-31-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<section id="rank-analysis" class="level3">
<h3 class="anchored" data-anchor-id="rank-analysis">Rank analysis</h3>
<p>If we rank our data this flattens spatial and genetic data such that we biologically are reducing the stronger effect on the edges. Of course this comes at a cost of reducing the magnitude those data points can pull. One could argue this may diminish our analysis (and underlying biological inference) of phylogenetic distance in the absence of spatial autocorrelation.</p>
<div class="cell">
<div class="cell-output-display">
<p><img src="analysis-showcase_files/figure-html/unnamed-chunk-32-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>If we run a rank correlation what happens is ND is strengthened but FD is reduced.</p>
<div class="cell">

</div>
<p>I also conducted a rank analysis to interspecific (as the genetic distance is so large) and optimistically our results are extremely robust across analyses. I have not included it here for brevity (document is already very long) but if people want to see it I am happy to show!</p>
</section>
</section>
<section id="reason-2-not-enough-time-for-competition." class="level2">
<h2 class="anchored" data-anchor-id="reason-2-not-enough-time-for-competition.">Reason 2: Not enough time for competition.</h2>
<p>Reason 2 may be that not enough time has occurred at early stages to ‘filter’ out lower fit genotypes.</p>
<p>While this is possible I do not believe this is true because we find no correspondence between FD and spatial distance. This lack of association suggests that even given time there is no reason to believe competitive interactions/periods of sympatry as a selective agent.</p>
<p>Of course current space does not indicate history of sympatry though. Perhaps more indicitive of history of sympatry is close genetic relatedness with high spatial distances?</p>
</section>
<section id="reason-3-selection-at-different-stages" class="level2">
<h2 class="anchored" data-anchor-id="reason-3-selection-at-different-stages">Reason 3: Selection at different stages</h2>
<p>Reason 3 would be based on the evolutionary rates we observe for niche and fitness differences. If it is to be believed ND evolves slower than FD, selection may favor greater FD at early stages. As ND accumulate, selection for greater FD may simply not be as strong. Personally I dont think we can use this argument by itself as its tautological with our own data. We would need to run a simulation which is outside the scope of this project but its a fun idea :)</p>
</section>
<section id="reason-4-not-capturing-moment-of-divergence" class="level2">
<h2 class="anchored" data-anchor-id="reason-4-not-capturing-moment-of-divergence">Reason 4: Not capturing moment of divergence</h2>
<p>This could be the most boring or most interesting answer depending who you ask. But we have few data points around origin and even though this experiment goes closer to the origin than ever before, we could still be missing something in the precise moment lineages diverge. Personally I find this hard to believe given the SNP data.</p>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->



</body></html>